win32/GNUmakefile

```diff
@@ -945,6 +945,7 @@ MICROCORE_SRC	=		\
 		..\dump.c	\
 		..\hv.c		\
 		..\av.c		\
+		..\builtin.c	\
 		..\caretx.c	\
```

Comment:
Adding a new C source file requires similar "cargo cult" in `vms/descrip_mms.template`

(`Makefile.SH`, both win32 makefiles, `vms/descrip_mms.template` and maybe I've missed somewhere else obscure)